tasks/notes.py

```diff
@@ -134,13 +134,29 @@ def update_changelog(ctx, release_branch, target="all", upstream="origin"):
                 code=1,
             )
 
+        # formats git output and retrieves branches from later
+        # releases (i.e. if we are on 7.67.2, and releases
+        # are being cut for 7.68.x and 7.69.x, they would be retrieved)
+        awk_script = (
+            f"awk -v min={new_version_int[1]} " + "{"
```

Comment:
This doesn't work on Windows. Since we are already using a programming language, can we just put the logic here?